src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx

```diff
@@ -42,36 +52,101 @@ Our AI has been trained to communicate with engineers using common questions and
         src={askgrok}
     />
 
-[NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/) is a powerful tool you can use to gain insights into systems, but when you're in a hurry you might not have time to create  the right query to get the data you need. If you're struggling to get a query to work, ask our AI to help you. You can ask questions like, "How is this service performing?" or "How has this service been performing over the past 24 hours?"
+[NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/) is a powerful tool you can use to gain insights into systems, but when you're in a hurry you might not have time to createthe right query to get the data you need. If you're struggling to get a query to work, ask our AI to help you. You can ask questions like, "How is this service performing?" or "How has this service been performing over the past 24 hours?"
 
 To enable NRAI in the NRQL console: 
 
 1. Go to the user menu at the bottom left of the New Relic UI.
 2.  Click the **Features** tab.
 3. Scroll down to the **New Relic AI** section and click **Enable**.
 
-## Sample questions [#sample-questions] 
+### Sample questions [#sample-questions] 
 
 You can chat with our AI to better understand every aspect of your data but if you're struggling with where to begin, here are some prompts. 
 
-### Get started with New Relic 
-Our AI chat experience can also help get you started using New Relic. If you need help organizing your data and setting up alerts, NRAI can act as a guide throughout your onboarding process. Ask our AI questions like:
+<Tabs>
```

Comment:
Our tab component does a really poor job at seperating the "tab" group from the rest of the content on the docs. 

Notice how in this screenshot you can't tell that `Get help analyzing error logs` isn't part of the tab. I'd move the tab group to the very bottom of the doc.

<img width="1137" alt="Screenshot 2024-01-04 at 4 19 31 PM" src="https://github.com/newrelic/docs-website/assets/42678939/d220a82c-635a-4e33-89b1-6c3566e7bd74">